tests/testthat/test-fmt_pad_num.R

```diff
@@ -1,15 +1,6 @@
-# Function to skip tests if Suggested packages not available on system
-check_suggests <- function() {
-  skip_if_not_installed("rvest")
-  skip_if_not_installed("xml2")
-  skip_on_cran()
-
-}
-
-
-
 test_that("gt_fmt_pad_num test that padding is correct", {
   check_suggests()
+  skip_on_cran()
```

Comment:
The few places where you do more than `check_suggests()`, I'm just in-lining the additional skips.